### ISSUE TITLE
Vert.x CRUD mission issues

### ISSUE THREAD

**Issue by cescoffier** [openshiftio/appdev-documentation#95]
1. You must be logged in beofre creating the database

The two sections are not in the right order.

2. `You app1 pod should have a status of Running once its fully deployed and started.` is not right

You should be running AND ready (1/1)

```
NAME                     READY     STATUS      RESTARTS   AGE
crud-vertx-1-deploy      1/1       Running     0          53s
crud-vertx-1-qrn65       0/1       Running     0          50s       <- running and not ready
crud-vertx-s2i-1-build   0/1       Completed   0          1m
http-vertx-1-qzttx       1/1       Running     0          17m         <- running and ready
http-vertx-s2i-1-build   0/1       Completed   0          18m
```

3. On my minishift `postgresql-ephemeral` is not available, I had to use `postgresql-persistent`

4. `oc get route vertx-crud -o jsonpath={$.spec.host}` should be `oc get route crud-vertx -o jsonpath={$.spec.host}`

It impacts all the output of the different commands

In addition, the output of the `oc get route....` is not right. It's just a single line: the DNS name:

```
oc get route crud-vertx -o jsonpath={$.spec.host}
crud-vertx-myproject.192.168.64.12.nip.io
```
5. The usage of `http` should be described somewhere. First the user needs to have httpie installed. Should we only use CURL ?

**Comment by rhoads-zach** [comment]
@inoxx03 can you take a look at this?

**Comment by inoxx03** [comment]
Fixes in PR #120. Please take a look. Thank  you
